setup.py

```diff
@@ -30,6 +30,13 @@
         'pytest-runner',
     ],
     classifiers=[
-        # TODO(Add items)
+        (
+            'License :: OSI Approved :: ',
+            'GNU General Public License v2 or later (GPLv2+)'
+        ),
```

Comment:
Thanks for mentioning it. I tried to do this http://python-guide-pt-br.readthedocs.io/en/latest/writing/style/#line-continuations . But I mistook it.
Rebased!